addons/binding/org.openhab.binding.loxone/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,111 @@
+<thing:thing-descriptions bindingId="loxone"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="miniserver">
+    
+        <label>Miniserver</label>
+        <description>www.loxone.de</description>
+        
+        <properties>
+            <property name="Serial number"></property>
+            <property name="Project name"></property>           
+            <property name="Miniserver name"></property>
+            <property name="Cloud address"></property>
+        </properties>
+        
+        <config-description>
+        
+            <parameter-group name="miniserver">
+                <label>Miniserver settings</label>
+                <description>Connection to Miniserver</description>
+                <advanced>false</advanced>
+            </parameter-group>
+
+            <parameter-group name="timeouts">
+                <label>Timeout settings</label>
+                <description>Various timeout parameters</description>
+                <advanced>true</advanced>
+            </parameter-group>
+                
+            <parameter-group name="sizes">
+                <label>Size settings</label>
+                <description>Various sizing parameters</description>
+                <advanced>true</advanced>
+            </parameter-group>
+
+            <parameter name="host" type="text" required="true" groupName="miniserver">
+                <label>Host</label>
+                <context>network_address</context>
+                <description>Host address or IP of the Loxone Miniserver</description>
+            </parameter>
+            <parameter name="port" type="integer" min="0" required="true" groupName="miniserver">
+                <label>Port</label>
+                <description>Web interface port of the Loxone Miniserver</description>
+            </parameter>
+            <parameter name="user" type="text" required="true" groupName="miniserver">
+                <label>User</label>
+                <description>User name on the Loxone Miniserver</description>
+            </parameter>
+            <parameter name="password" type="text" required="true" groupName="miniserver">
+                <label>Password</label>
+                <context>password</context>
+                <description>User password on the Loxone Miniserver</description>
+            </parameter>
+            
+            <parameter name="firstConDelay" type="integer" min="0" max="120" required="true" groupName="timeouts">
+                <label>First connection delay</label>
+                <description>Time between binding initialization and first connection attempt (seconds, 0-120)</description>
+                <advanced>true</advanced>
```

Comment:
Empirically, setting advanced=true on at least one of {group, parameter} produces the same desired result: parameter and its group end up in the hidden section on the UI.
If this is by design enough to have it on group only and omit on parameters, I would remove them from parameters due to simplicity.